ap-ping-update

AndrePanin opened the pull request:
Updated after https://github.com/gear-tech/apps/pull/22

EugenWay commented:
Why you added logger and test if not minimal state required?
We deleted it a few days ago.

shamilsan commented on docs/getting-started-in-5-minutes.md:
````diff
@@ -116,6 +155,10 @@ This simple smart-contract responds with `PONG` to a `PING` message sent to the
 ```bash
 cd ~/gear/contracts/first-gear-app/
 RUSTFLAGS="-C link-args=--import-memory" cargo +nightly build --release --target=wasm32-unknown-unknown
+
````
Need to remove it.
```suggestion
```

breathx replied:
Yeah, `gear-wasm-builder` puts everything instead of you. So `cargo build` and `cargo test` or `cargo test --release` are the only required commands

shamilsan reviewed (approved):
Minor issues to be fixed.

breathx commented on docs/getting-started-in-5-minutes.md:
```diff
@@ -80,32 +88,63 @@ authors = ["Your Name"]
+        assert!(res.log().is_empty());
+
+        let res = program.send_bytes(42, "PING");
+        assert!(res.contains(&Log::builder().source(1).dest(42).payload_bytes("PONG")));
```
I would separate `Log` build into some var to increase the readability

AndrePanin replied:
ok, removed from getting started. in examples will be updated after change in /apps/
https://github.com/gear-tech/apps/blob/master/ping/src/lib.rs

breathx reviewed (approved):
Need to resolve opened conversations and would be good!

shamilsan commented:
> Why you added logger and test if it is not minimal state? We deleted it a few days ago.

We need to decide whether we provide a super minimal example, or just copy `demo-ping` here.

AndrePanin commented:
> > Why you added logger and test if it is not minimal state? We deleted it a few days ago.
> 
> We need to decide whether we provide a super minimal example, or just copy `demo-ping` here.

ok, removed test from getting started, remained in examples
